src/components/Reviews.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import reviews from '../mocks/reviews.json'
+
+export default function Reviews() {
+  return (
+    <>
+      {reviews.map((review, index) => (
+        <Review key={index}>
+          <h2>{review.title}</h2>
+          <p>Rating: {review.rating}</p>
+          <h3>{review.categorie}</h3>
+          <h3>{review.subcategorie}</h3>
+          <h3>Summary: </h3>
+          <p>{review.summary}</p>
+          <h3>Lessons: </h3>
+          <p>{review.lessons}</p>
```

Comment:
Not shure if it be better to use h4 for review.subcategorie, or p is missing after h3 review.categorie
looks odd

Reply:
You are right, changed it!